apps/src/netsim/DashboardUser.js

```diff
@@ -91,7 +91,9 @@ DashboardUser.getCurrentUser = function () {
         DashboardUser.currentUser_.initialize_(data);
       },
       error: function (/*jqXHR, textStatus, errorThrown*/) {
-        throw new Error("Unable to retrieve current user info.");
+        DashboardUser.currentUser_.initialize_({
```

Comment:
Here, a 'static' method is accessing a private method on an instance of its own class.  I think that's usually allowed.  Do we prefer to avoid this?